### ISSUE TITLE
Dungeon Town game requires original base game?

### ISSUE THREAD

**Issue by SilentZeroFool** [MachineGunnur/GOG-Games#344]
Im not sure why it needs the base game when this is the base game just unrated version. The installer is calling this version a DLC. Do you need the original rated version?
![image_2021-12-05_165738](https://user-images.githubusercontent.com/95578307/144751864-4112bd1d-e6e3-40e5-b1fc-786b99f0b9b6.png)

**Comment by Masamune3210** [comment]
![image](https://user-images.githubusercontent.com/1053504/144757897-02a2b6cb-acde-4c90-8398-03a07271760a.png)

**Comment by SilentZeroFool** [comment]
> ![image](https://user-images.githubusercontent.com/1053504/144757897-02a2b6cb-acde-4c90-8398-03a07271760a.png)

Yes but the one on the site is called Dungeon Town and the gog link is linked to Dungeon Town as well and not Dungeon Town UNRATED. That's where I got confused. Only the file itself had the word unrated in it so never thought it was actually a DLC I was downloading instead of the actual game. But its my fault for not investigating and instead I downloaded it thinking it is what it is as shown.
